lib/client.js

```diff
@@ -48,10 +49,12 @@ function timeoutReject (client, reject, name, paramsArray) {
 }
 
 function defaultTimer (client, actions, callback) {
-  return setTimeout(() => {
+  const timer = new PausableTimer(() => {
     trackSDKRequestTimeout(client, actions, PROMISE_TIMEOUT_LONG)
     callback(new Error('Invocation request timeout'))
   }, PROMISE_TIMEOUT_LONG)
+  requestTimers.push(timer)
```

Comment:
Just realised this won't work since the completed timers will need to be removed from this queue or they'll just start all over when we revisit the page a second time - I'll fix this.